test/appium/tests/atomic/chats/test_public.py

```diff
@@ -328,6 +329,304 @@ def test_public_chat_open_using_deep_link(self):
+        profile_2.switch_network()
+        [home.home_button.click() for home in (cls.home_1, cls.home_2)]
+        cls.chat_2 = cls.home_2.add_contact(cls.public_key_1)
+        cls.chat_2.chat_message_input.send_keys('test message')
+        cls.chat_2.send_message_button.click()
+        cls.home_2.just_fyi('Install free sticker pack and use it in 1-1 chat')
+        cls.chat_2.install_sticker_pack_by_name()
+        cls.chat_1 = cls.home_1.add_contact(cls.public_key_2)
+        [home.home_button.click() for home in (cls.home_1, cls.home_2)]
+        cls.group_chat_name = "GroupChat"
+        cls.group_chat_1 = cls.home_1.create_group_chat(user_names_to_add=[cls.default_username_2], group_chat_name=cls.group_chat_name)
+        cls.home_2.get_chat(cls.group_chat_name).click()
+        cls.group_chat_2 = cls.home_2.get_chat_view()
+        cls.group_chat_2.join_chat_button.click()
+        [home.home_button.click() for home in (cls.home_1, cls.home_2)]
+        cls.public_chat_name = cls.home_1.get_random_chat_name()
+        cls.public_chat_1, cls.public_chat_2 = cls.home_1.join_public_chat(cls.public_chat_name), cls.home_2.join_public_chat(
+            cls.public_chat_name)
+        [home.home_button.click() for home in (cls.home_1, cls.home_2)]
+        cls.home_1.get_chat(cls.default_username_2).click()
+        cls.home_2.get_chat(cls.default_username_1).click()
+        cls.message_1, cls.message_2, cls.message_3, cls.message_4 = "Message1", "Message2", "Message3", "Message4"
+
+    @marks.testrail_id(702065)
+    @marks.medium
+    def test_messaging_markdown_support_in_messages(self):
+
+        markdown = {
+            'bold text in asterics': '**',
+            'bold text in underscores': '__',
+            'italic text in asteric': '*',
+            'italic text in underscore': '_',
+            'inline code': '`',
+            'code blocks': '```',
+            'quote reply (one row)': '>',
+        }
+
+        for message, symbol in markdown.items():
+            self.home_1.just_fyi('checking that "%s" is applied (%s) in 1-1 chat' % (message, symbol))
+            message_to_send = symbol + message + symbol if 'quote' not in message else symbol + message
+            self.chat_2.chat_message_input.send_keys(message_to_send)
+            self.chat_2.send_message_button.click()
+            if not self.chat_2.chat_element_by_text(message).is_element_displayed():
+                self.errors.append('%s is not displayed with markdown in 1-1 chat for the sender \n' % message)
+
+            if not self.chat_1.chat_element_by_text(message).is_element_displayed():
+                self.errors.append('%s is not displayed with markdown in 1-1 chat for the recipient \n' % message)
+
+        [chat.home_button.double_click() for chat in (self.chat_1, self.chat_2)]
+        chat_name = self.home_1.get_random_chat_name()
+        [home.join_public_chat(chat_name) for home in (self.home_1, self.home_2)]
```

Comment:
no reason to join to public chat 2d time, as already joined in setup_method

Reply:
changed to [home.get_chat(self.public_chat_name) for home in (self.home_1, self.home_2)]